app/javascript/blacklight/button_focus.js

```diff
@@ -0,0 +1,9 @@
+Blacklight.onLoad(function() {
+  // Button clicks should change focus. As of 10/2/19, Firefox for Mac and
+  // Safari both do not do this correctly.
+  document.addEventListener('click', event => {
+    if (event.target.matches('button')) {
```

Comment:
One last question.. should we only select the facets in question here? Or is it ok that this is happening globally?

@jcoyne any thoughts?

Reply:
Yeah, I'm not keen on checking every click. can you do `document.querySelector(...).forEach((elem) => elem.addEventListener())`